### DISCUSSION TITLE
bankacoount code

### DISCUSSION THREAD

**Discussion by Mallikarjunahugar** [akash-coded/spring-framework#66] [Q&A]
package bank;
import java.util.*;
public class Bankaccount {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
 int ac_no;
 String name, type;
 float amount;
	}
 void details(int n,String nm,String t, float a)
 {
	 ac_no=n;
	 name=nm;
	 type=t;
	 amount=a;
	  }
 void show()
 {
	 int name;
	System.out.println(ac_no+" ",+name+" ",+type+" ",+amount);
 }
 void deposit(float a)
 {
	 amount+=a;
	 System.out.println("deposit amount");
	  }
 void withdraw(float a)
 {
	 if(amount<a)
	 {
		 System.out.println("Insufficient amount");
	 
		 else
			 System.out.println("withdraw an amount:"+a);
		 }
	 void checkbalance()
	 {
		 System.out.println("balance is:"+amount);
	 }
 }
 class test{
	 {
		 bank b=new bank();
		 b.details(123456,"Arjuna","Saving account",1001);
		 b.show();
		 b.deposit(2001);
		 b.checkbalance();
		 b.withdraw(1500);
		 b.checkbalance();
		 }
 }

**Comment by akash-coded** [comment]
Please submit it under the appropriate discussion and then you can also delete this solo discussion from here. @Mallikarjunahugar
